test/functional/feature_mnehf.py

```diff
@@ -251,8 +245,8 @@ def run_test(self):
 
         ehf_tx_new_start = self.create_mnehf(28, pubkey)
 
-        self.log.info("activate MN_RR also by enabling spork 24")
-        assert_equal(get_bip9_details(node, 'mn_rr')['status'], 'defined')
+        self.log.info("Test spork 24 (EHF)")
+        assert_equal(get_bip9_details(node, 'testdummy')['status'], 'defined')
```

Comment:
```suggestion
        self.check_fork('defined')
```

should also drop
```
        self.log.info(f"bip9: {get_bip9_details(node, 'mn_rr')}")
        assert_equal(get_bip9_details(node, 'mn_rr')['status'], 'active')
```
at the end of the test